docs/user-guide/README.md

```diff
@@ -0,0 +1,344 @@
+```
+- In a similar way, PVC with `volumeMode: Block` can also be created and
+  claimed by App pod as below
+``` yaml
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: lvm-block-pvc
+spec:
+  volumeMode: Block
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+  storageClassName: topolvm-vg1
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: app-block
+spec:
+  containers:
+  - name: app-block
+    image: bash
+    imagePullPolicy: IfNotPresent
+    command: ["/usr/local/bin/bash", "-c", "/usr/bin/tail -f /dev/null"]
+    volumeDevices:
+    - devicePath: "/dev/xvda"
+      name: lvm-block-pvc
+  volumes:
+    - name: lvm-block-pvc
+      persistentVolumeClaim:
+        claimName: lvm-block-pvc
+EOF
+```
+- Status of both PVCs and App pods can be referred as below
+``` console
+kubectl get pod | grep -E '^(app|NAME)'
+NAME                                  READY   STATUS    RESTARTS   AGE
+app-block                             1/1     Running   0          99s
+app-file                              1/1     Running   0          9m34s
+
+kubectl get pvc
+NAME            STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+lvm-block-pvc   Bound    pvc-56fe959a-1184-4fa6-9d66-b69c06d43cf4   5Gi        RWO            topolvm-vg1    104s
+lvm-file-pvc    Bound    pvc-04a4f5f7-8665-4008-818a-490503c859f5   5Gi        RWO            topolvm-vg1    13m
+```
+
+### Monitoring and Metrics
+
+- LVMO currently surfaces only Topolvm metrics and those can be viewed either
+  from UI or port-forwarding the service
+- Make sure the namespace LVMO is running in has below label or patch the
+  namespace for metrics to be available in UI
+``` console
+kubectl patch namespace/lvm-operator-system -p '{"metadata": {"labels": {"openshift.io/cluster-monitoring": "true"}}}'
+```
+``` console
+# port-forward service in one terminal
+kubectl port-forward svc/topolvm-node-metrics :8080
+Forwarding from 127.0.0.1:41685 -> 8080
+Forwarding from [::1]:41685 -> 8080
+...
+...
+
+# in another terminal view the metrics at above port in localhost
+curl -s 127.0.0.1:41685/metrics | grep -Ei 'topolvm_volumegroup_.*?_bytes\{'
+topolvm_volumegroup_available_bytes{device_class="vg1",node="rhocs-bm4.lab.eng.blr.redhat.com"} 4.790222323712e+12
+topolvm_volumegroup_size_bytes{device_class="vg1",node="rhocs-bm4.lab.eng.blr.redhat.com"} 4.800959741952e+12
+```
+
+## Cleanup
+
+- Feature to auto cleanup volume groups after removing LVMCluster CR is coming
+  soon
+- Until then, please follow below procedure to cleanup left out resources
+  during CR deletion
+
+### Steps
+
+1. Make sure no PVC is bound to an App pod and remove the PVCs, all app pods can be removed in parallel and corresponding PVCs can be removed after pod deletion, below deletion is only for representation
```

Comment:
Simplify the sentence. Suggestion:
Remove all apps using PVCs created with topolvm.